Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -1278,17 +1278,17 @@ static _WKAutoplayEvent toWKAutoplayEvent(WebCore::AutoplayEvent event)
     if (!needsFontAttributes())
         return;
 
-    auto privateUIDelegate = (id<WKUIDelegatePrivate>)uiDelegate->m_delegate.get();
+    RetainPtr privateUIDelegate = (id<WKUIDelegatePrivate>)uiDelegate->m_delegate.get();
```

Comment:
Seems like could use a helper function for this so we don’t have to have these type casts all over this file?